CMakeLists.txt

```diff
@@ -29,8 +29,17 @@ add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_FULL_BINDIR}"
                           INSTALL_NAME_DIR "${CMAKE_INSTALL_FULL_LIBDIR}"
                           USE_LINK_PATH)
 
+find_package(YARP REQUIRED)
+find_package(IWear 1.9.0 REQUIRED)
+find_package(Eigen3 REQUIRED)
+
+
+set(YARP_FORCE_DYNAMIC_PLUGINS ON)
```

Comment:
Naive question, it is not clear to me where `YARP_FORCE_DYNAMIC_PLUGINS` is used

Reply:
It was needed to make sure that the devices was visible by ``yarp`` when doing ``yarpdev --list``. In fact, when enabling this option, an ``ini`` file is automatically generated in the ``/yarp/plugins`` folder, allowing ``yarp`` to load this device dynamically. This is necessary, otherwise, if the device was static, we would have needed to compile ``yarp`` with it, so that when running ``yarprobotinterface`` it would be able to find it.